input/includes/rrv-dev-note.md

```diff
@@ -0,0 +1,4 @@
+<div class="stu-note" markdown="1">
+ The [RANZCR Radiology Procedures](ValueSet-ranzcr-radiology-procedures.html) value set is  for Trial Use and will be built on during the course of Implementation Guide development. The ValueSet is currently comprised of X-Ray, Angiography and Ultrasound single code concepts from SNOMED CT AU. Once balloted, the RANZCR Radiology Procedures Value Set will be based in the
```

Comment:
I think for the purposes of the IG we need to shift the language and focus of the STU Note a bit. We can't use the term "Trial Use" in the context of this artefact in this standards development space at this time. If it is of benefit to incorporate the activities of trial use then put that into sentences and clarify how and when feedback is to come in. Remove the wording on balloting.

If the value set currently contains. What is coming? When are the timeframes for that?

1. Suggest wording can be more:

> Work is underway within RANZCR to define a Radiology Referral Reference Set. The [RANZCR Radiology Procedures](ValueSet-ranzcr-radiology-procedures.html) value set included in this guide is an evolving draft that supports the latest agreed set of concepts from the Radiology Referral Reference Set. At this time the value set contains the concepts for  X-Ray, Angiography and Ultrasound.  Implementers are advised to take note that concepts within the value set will continue to expand and evolve when further iterations of the Radiology Referral Reference Set are available.

2. The STU Note on StructureDefinition-au-erequesting-servicerequest-imag-intro.md needs to be replaced with this STU note.

3. If there are any _specific_ requests for feedback then they should be defined in additional STU Note content.

Reply:
I have addressed 1 and 2 with the suggested wording in an intro.md file.